securitycenter/docs/snippets_list_assets.py

```diff
@@ -17,20 +17,24 @@
 """ Examples of listing assets in Cloud Security Command Center."""
 import os
 from datetime import datetime, timedelta
+import pytest
 
 
-# The numeric identifier for the organization.
-ORGANIZATION_ID = os.environ["GCLOUD_ORGANIZATION"]
+@pytest.fixture(scope="module")
+def organization_id():
+    """Get Organization ID from the environment variable """
+    return os.environ["GCLOUD_ORGANIZATION"]
 
 
-def test_list_all_assets():
+def test_list_all_assets(organization_id):
     """Demonstrate listing and printing all assets."""
-    from google.cloud import securitycenter_v1beta1 as securitycenter
-
     # [START demo_list_all_assets]
+    from google.cloud import securitycenter as securitycenter
```

Comment:
"from x import y as y" does the same thing as just "from x import y". So all these lines can just be "from google.cloud import securitycenter".

Reply:
done.

Reply:
done (will push latest commit in a minute)